TACC/Core-Portal
# bug/FP-1059: Fix "Shared Workspaces" Breadcrumb in File Selector Modal

## Overview: ##

"Shared Workspaces" Breadcrumb in File Selector Modal links to Data Files

## Related Jira tickets: ##

* [FP-1059](https://jira.tacc.utexas.edu/browse/FP-1059)

## Summary of Changes: ##

Breadcrumb root link in File Selector Modal should remain in the modal, not routing to Data Files section.

## Testing Steps: ##
1. Open File Select modal (eg. Applications -> {APPLICATION} -> Input directory -> Select)
2. Select input -> Shared workspaces
3. Navigate into a workspace
4. Click root breadcrumb (Shared workspaces)
5. Modal window should be open still and displaying projects listing

## UI Photos:

<img width="888" alt="Screen Shot 2021-08-26 at 2 34 55 PM" src="https://user-images.githubusercontent.com/62672123/131025234-f8789363-b0dc-487d-b396-105d08399e99.png">


## Notes: ##
